pom.xml

```diff
@@ -53,7 +53,7 @@
   </developers>
 
   <prerequisites>
-    <maven>${mavenVersion}</maven>
+    <maven>${maven.version}</maven>
```

Comment:
ups .... I know I have introduced this property ... but now I think it was wrong the same property is set by Maven itself 😞 
I'm still learning 😄